io-engine/src/bdev/nexus/nexus_bdev_children.rs

```diff
@@ -815,37 +816,70 @@ impl<'n> Nexus<'n> {
         child_device: &str,
         reason: FaultReason,
         retry: bool,
-    ) {
+    ) -> Option<RebuildLogHandle> {
         let name = self.name.clone();
 
-        if let Some(c) = self.as_mut().lookup_child_by_device_mut(child_device)
+        let Some(c) = self.as_mut().lookup_child_by_device_mut(child_device) else {
```

Comment:
Ah I thought let else was still unstable, cool then maybe we can now also use this in control-plane